pkg/costmodel/metrics.go

```diff
@@ -730,17 +733,21 @@ func (cmme *CostModelMetricsEmitter) Start() bool {
 			for labelString, seen := range containerSeen {
 				if !seen {
 					labels := getLabelStringsFromKey(labelString)
-					ok := cmme.RAMAllocationRecorder.DeleteLabelValues(labels...)
-					if !ok {
-						log.Errorf("Metric emission: failed to delete RAMAllocation with labels: %v", labels)
-					}
-					ok = cmme.CPUAllocationRecorder.DeleteLabelValues(labels...)
-					if !ok {
-						log.Errorf("Metric emission: failed to delete CPUAllocation with labels: %v", labels)
-					}
-					ok = cmme.GPUAllocationRecorder.DeleteLabelValues(labels...)
-					if !ok {
-						log.Errorf("Metric emission: failed to delete GPUAllocation with labels: %v", labels)
+					if labels[1] == unmountedPVsContainer { // special "pod" to contain the unmounted PVs - does not have RAM/CPU/...
```

Comment:
Why are you proposing this conditional?

Reply:
Or perhaps this is meant to be `!=`, so that we stop attempting to delete the fake allocation data?

Reply:
Totally my fault - tried to clean the code after validating, the statement got reversed wrongly. It is indeed a negated comparison I'd wanted there (`!=`). Changed it as it should be.